src/webview/js/js.js

```diff
@@ -624,7 +617,7 @@ const handleLongPress = (target: Element) => {
   sendMessage({
     type: 'longPress',
     target: target.matches('.header') ? 'header' : 'message',
-    messageId: getMessageIdFromNode(target),
+    messageId: ancestorMessageId(target),
```

Comment:
I'd be curious to see if that will not throw the exception in `ancestorMessageId ` because of the `if (!messagePeerAncestor) {` clause.

Reply:
Thanks!

Moved this code over to #3097 .